ansible/roles/postgresql-server/tasks/install-Ubuntu.yml

```diff
@@ -56,6 +56,11 @@
     - "postgresql-contrib-9.6"
     - "postgresql-plperl-9.6"
     - "postgresql-server-dev-9.6"
+    - "postgresql-10"
+    - "postgresql-client-10"
```

Comment:
should this be postgresql-11?

Reply:
That's the Ansible task that removes old versions of PostgreSQL before installing the designated version (11 in this case).